### PR TITLE
Add color and emotion study

### PR DESCRIPTION
**Pull request by AnnikaTjuka**
# Pull request checklist

- [X] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

This PR is based on #928. It adds two lists a) Scherer-2013-20 with the Geneva emotion wheel and b) Jonauskaite-2020-12 with color terms across 22 languages.

### CODE REVIEW

**Comment by AnnikaTjuka** [comment]
@chrzyki Would you mind moderating this PR? The lists are rather small (12 and 20 items), so one review might be sufficient.

File: concepticondata/conceptlists/Jonauskaite-2020-12.tsv-metadata.json [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                    {
+                        "datatype": "string",
+                        "name": "ENGLISH"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "ARABIC"
+                    },
+                    {
+                        "datatype": "string",
```
Comment (chrzyki) [inline concepticondata/conceptlists/Jonauskaite-2020-12.tsv-metadata.json:51]: Language columns could be linked to Glottolog with a `propertyUrl`.

File: concepticondata/concepticon.tsv [hunk 2]
```diff
@@ -3897,3 +3897,6 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3911	COW MILK	Food and drink	The milk of cows.	Person/Thing
 3912	BREAST MILK	The body	The milk of humans.	Person/Thing
 3913	COUGH (ACT OF COUGHING)	Basic actions and technology	The act of coughing or the sound of coughing.	Person/Thing	
+3914	TURQUOISE	Sense perception	Having a light greenish blue as its color.	Property	
+3915	PRIDE	Emotions and values	An emotional state of being proud.	Person/Thing
```
Comment (chrzyki) [inline concepticondata/concepticon.tsv:3901]: Could also be done in this PR I think (if you don't mind):

Link:

- Hale-1973-1798-1116
- Lynott-2013-400-342
- Hill-2015-999-552
- Miklashevsky-2018-506-79
- Zalizniak-2020-2591-2244

File: concepticondata/concepticon.tsv [hunk 3]
```diff
@@ -3897,3 +3897,6 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3911	COW MILK	Food and drink	The milk of cows.	Person/Thing
 3912	BREAST MILK	The body	The milk of humans.	Person/Thing
 3913	COUGH (ACT OF COUGHING)	Basic actions and technology	The act of coughing or the sound of coughing.	Person/Thing	
+3914	TURQUOISE	Sense perception	Having a light greenish blue as its color.	Property	
+3915	PRIDE	Emotions and values	An emotional state of being proud.	Person/Thing	
+3916	PLEASURE	Emotions and values	An emotional state of sensual gratification or frivolous amusement.	Person/Thing
```
Comment (chrzyki) [inline concepticondata/concepticon.tsv:3902]: Could also be done in this PR I think (if you don't mind):

Link:

- Pereira-2018-180-122
- Miklashevsky-2018-506-435
- Zalizniak-2020-2591-1490

File: concepticondata/concepticon.tsv [hunk 4]
```diff
@@ -3897,3 +3897,6 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3911	COW MILK	Food and drink	The milk of cows.	Person/Thing
 3912	BREAST MILK	The body	The milk of humans.	Person/Thing
 3913	COUGH (ACT OF COUGHING)	Basic actions and technology	The act of coughing or the sound of coughing.	Person/Thing	
+3914	TURQUOISE	Sense perception	Having a light greenish blue as its color.	Property
```
Comment (chrzyki) [inline concepticondata/concepticon.tsv:3900]: Could also be done in this PR I think (if you don't mind):

Link:

- Morucci-2019-643-604

**Comment by xrotwang** [comment]
Regarding language columns: these are aggregated in
concepticondata/concepticon.json and could be annotated there.

Christoph Rzymski ***@***.***> schrieb am Do., 27. Mai 2021,
10:40:

**Comment by AnnikaTjuka** [comment]
@xrotwang So I would add the languages that are not listed yet (e.g., Georgian) to the concepticon.json. Is that correct?
@chrzyki I had planned to do the additional mappings in a separate PR, but it makes sense to do it now.

**Comment by xrotwang** [comment]
Yes, adding language columns to concepticon.json would be good.

Annika Tjuka ***@***.***> schrieb am Do., 27. Mai 2021, 10:53:

**Comment by AnnikaTjuka** [comment]
Ok, will do!

**Comment by chrzyki** [comment]
> 
> 
> @xrotwang So I would add the languages that are not listed yet (e.g., Georgian) to the concepticon.json. Is that correct?
> @chrzyki I had planned to do the additional mappings in a separate PR, but it makes sense to do it now.

Yes, thanks! And sorry for pushing the additional mappings - I think it would be okay to have them in this PR as well.

**Comment by AnnikaTjuka** [comment]
Languages and additional mappings are updated.

**Review by LinguList — APPROVED** [review]
Very nice to add this list, and to map past concepts!

**Review by chrzyki — APPROVED** [review]

**Comment by AnnikaTjuka** [comment]
Thanks for the approvals! I'll merge this now.

**Pull request merged**